Packs/jamf/README.md

```diff
@@ -0,0 +1,14 @@
+This pack enables you to remotely manage Apple devices (Mac, iPhone, Apple TV, iPad). It can be used to control

```

Comment:
@barchen1 @ShirleyDenkberg  I think that this description is not quite accurate since this pack won't let you to manage the devices (although it has several commands which will allow you to block/erase certain devices). The pack will allow you to interact with "Jamf Pro" which is the product that allows you to manage the devices.

Reply:
@tomer-pan Thanks.  I updated the description.